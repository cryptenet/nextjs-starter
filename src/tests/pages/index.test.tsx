import IndexPage from '@pages/index';
import { render } from '../testUtils';

describe('Home', () => {
    it('matches snapshot', () => {
        const { asFragment } = render(<IndexPage />, {});
        expect(asFragment()).toMatchSnapshot();
    });
});
