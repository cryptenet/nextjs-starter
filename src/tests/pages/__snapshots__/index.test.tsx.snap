// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home matches snapshot 1`] = `
<DocumentFragment>
  <main
    class="flex flex-col justify-center items-center h-screen"
  >
    <header
      class="flex flex-col w-screen"
    >
      <nav
        class="container"
      />
    </header>
    <section
      class="container flex flex-col flex-1 justify-center items-center w-screen"
    >
      <div
        class="w-4/5 h-1/6"
        style="opacity: 0;"
      >
        <div
          class="w-full h-full bg-brand-logo bg-no-repeat bg-center bg-contain"
        />
      </div>
      <h1
        class="text-lg sm:text-2xl md:text-4xl mt-8"
        style="opacity: 0;"
      >
        Next.js Starter Template
      </h1>
    </section>
    <footer
      class="flex flex-col w-screen pt-8 pb-6"
    >
      <p
        class="text-center text-sm"
      >
        Â© Cryptenet Limited 2018-2023
      </p>
    </footer>
  </main>
</DocumentFragment>
`;
